specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -7118,7 +7118,7 @@
         },
         "supportAutomaticPlacement": {
           "type": "boolean",
-          "description": "Specifies whether virtual machines or virtual machine scale sets can be placed automatically on the dedicated host group. Automatic placement means resources are allocated on dedicated hosts, that are chosen by Azure, under the dedicated host group. The value is defaulted to 'true' when not provided. <br><br>Minimum api-version: 2020-06-01."
+          "description": "Specifies whether virtual machines or virtual machine scale sets can be placed automatically on the dedicated host group. Automatic placement means resources are allocated on dedicated hosts, that are chosen by Azure, under the dedicated host group. The value is defaulted to 'false' when not provided. <br><br>Minimum api-version: 2020-06-01."
```

Comment:
although this isn't a Swagger breaking change, but service behavior breaking change. what's service side plan to communicate the change to end users? It's better to call out this breaking behavior change in downstream SDKs, Client tools with service communication together.

Reply:
This feature has been under a limited preview only for whitelisted subscriptions with no SLA guarantee, and only few subscriptions have signed up, so the impact should be small. Also, we will have a blog post when announcing GA.